test/test-notification-model-updates.js

```diff
@@ -519,6 +534,24 @@ test('Test notifications', function(t) {
     });
   });
 
+  t.test('check change stream', function(tt) {
+    var change = changeStream.read();
+    tt.equal(change.target, 'ServiceInstance-1');
+    tt.equal(change.type, 'create');
+    change = changeStream.read();
+    tt.equal(change.target, 'ServiceInstance-1');
+    tt.equal(change.type, 'update');
+    change = changeStream.read();
+    var eventCnt = 0;
+    while (change !== null) {
+      eventCnt++;
+      change = changeStream.read();
+    }
+    tt.equal(eventCnt, 21);
```

Comment:
What is the significance of this `21`? An assertion description would be particular appropriate here so that if the tests fail at some point because there are 20 or 22 someone will have a hope of figuring out what that means.